src/main/java/ro/sdaboys/restaurantapp/dto/ReservationTimeDto.java

```diff
@@ -0,0 +1,34 @@
+package ro.sdaboys.restaurantapp.dto;
+
+import java.time.LocalDateTime;
+
+public class ReservationTimeDto {
+
+    private Long id;
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public LocalDateTime getStartTime() {
+        return startTime;
+    }
+
+    public void setStartTime(LocalDateTime startTime) {
+        this.startTime = startTime;
+    }
+
+    public LocalDateTime getEndTime() {
+        return endTime;
+    }
+
+    public void setEndTime(LocalDateTime endTime) {
```

Comment:
Since we are using lomblok, instead of making all getters and setters we can use @Data